CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ vNext
 
 ### Features
 
+- Implemented the search functionality to the Dropdown component ([PR 165](https://github.com/input-output-hk/daedalus/pull/165))
```

Comment:
I think it would be good to be more specific here, as the search functionality was added to the `Options` and not `Dropdown` 😉

Reply:
True! Done :)